run_dir/design/flowcell.html

```diff
@@ -26,7 +26,19 @@
       </tr>
       <tr>
         <th>Projects:</th>
-        <td>{{ ', '.join(sorted(flowcell.get('plist', []))) }} </td>
+        <td>
+          {% for project in sorted(flowcell.get('plist', [])) %}
+          <div class="btn-group btn-group-sm">
+            <button type="button" class="btn btn-sm btn-default dropdown-toggle"
+                    data-toggle="dropdown"
+            >{{ project }}&nbsp;<span class="caret"></span></button>
```

Comment:
I know you don't like long lines in your Python scripts, but I try to avoid breaking single HTML tags over lines where possible. Perhaps this instead?

```html
<button type="button" class="btn btn-sm btn-default dropdown-toggle" data-toggle="dropdown">
    {{ project }}&nbsp;<span class="caret"></span>
</button>
```